Makefile

```
@@ -456,26 +456,19 @@ clean:
 $(DOCKERFILE_FROM_CHECKER): $(DOCKERFILE_FROM_CHECKER)
 	make -C $(DOCKERFILE_FROM_CHECKER_DIR)
 
+# this next section checks that the FROM hashes for any image in any dockerfile anywhere here are consistent.
+# For example, one Dockerfile has foo:abc and the next has foo:def, it will flag them.
+# These are the packages that we are ignoring for now
+IGNORE_HASHES_PKGS=bsp-imx vtpm optee-os installer wwan wlan watchdog uefi acrn acrn-kernel u-boot udev xen-tools xen alpine
+IGNORE_HASHES_TOOLS=bpftrace-compiler
```

What's the difference between a "package" and a "tool" in this case?

---

Different directories.

---

```suggestion
IGNORE_HASHES_EVE_TOOLS=bpftrace-compiler
```
who knows when we will have `Dockerfiles` in the `tools` directory

---

Good call. Updated.